synapse/config/repository.py

```diff
@@ -36,6 +38,10 @@
     "ThumbnailRequirement", ["width", "height", "method", "media_type"]
 )
 
+MediaStorageProviderConfig = namedtuple(
```

Comment:
can we document the members on here somehow, please?

Reply:
ok thanks. for future ref, I'd like to see their types made explicit too, but this is fine for now

Reply:
Ah, yup, oops.